frontend/apps/reader/modules/readerlink.lua

```diff
@@ -790,7 +790,9 @@ function ReaderLink:onGotoLink(link, neglect_current_location, allow_footnote_po
             if not neglect_current_location then
                 self:addCurrentLocationToStack()
             end
+            local originating_page = self.ui:getCurrentPage()
```

Comment:
For paging, it is `self.paging.current_page`.

Reply:
Sorry, I'm not sure what you mean. Assigning `self.paging.current_page` doesn't work, it returns a nil value.

Reply:
`self.ui.paging.current_page`